lua/r/send.lua

```diff
@@ -105,15 +105,16 @@ local function get_code_to_send(txt, row)
     if vim.o.filetype == "rhelp" then return get_rhelp_code_to_send(txt, row) end
 
     -- Find the 'root' node for the current expression --------------------
-    local n_lang = vim.o.filetype == "rnoweb" and "r" or nil
     local node = vim.treesitter.get_node({
         bufnr = 0,
         pos = { row - 1, col - 1 },
-        lang = n_lang,
         -- Required for quarto/rmd/rnoweb; harmless for r.
         ignore_injections = false,
     })
 
+    -- This is a strange fix for https://github.com/R-nvim/R.nvim/issues/298
+    if node then vim.schedule(function() end) end
```

Comment:
I do not understand this code, but apparently it is strange enough if fixes the issues :)

Reply:
I guess it fixes a Neovim bug. Compared to Vim, Neovim seems to be much more optimized to avoid unnecessary computations of changes in the user interface while commands are running. It only updates the user interface when the execution of commands finish. My guess is that something needs to be updated immediately and the call to `vim.schedule()` forces the necessary update. It's just a guess.

Reply:
I see. LGTM then.